reference/docs-conceptual/components/vscode/using-vscode.md

```diff
@@ -23,47 +28,48 @@ macOS, and Linux, see the following links:
 
 For traditional Windows PowerShell workloads, see [Installing Windows PowerShell][install-winps].
 
-## Editing with VSCode
+> [!NOTE]
+> Visual Studio Code is not the same as [Visual Studio](https://visualstudio.microsoft.com/).
 
-1. Install VSCode. For more information, see the overview
-   [Setting up Visual Studio Code](https://code.visualstudio.com/Docs/setup/setup-overview).
+> [!IMPORTANT]
+> The [Windows PowerShell ISE][ise] is also still available for Windows, however, it is no longer in
```

Comment:
Maybe worth mentioning here that the ISE doesn't support PS 6 or 7

Reply:
Added